tests/test_helpers.py

```diff
@@ -1042,20 +1042,13 @@ async def test_image_file_name(
 
 async def test_amazon_exception(hass, mock_imap_amazon_exception, caplog):
     result = amazon_exception(mock_imap_amazon_exception, ['""'])
-    assert result["order"] == [
-        "123-1234567-1234567",
-        "123-1234567-1234567",
-        "123-1234567-1234567",
-        "123-1234567-1234567",
-        "123-1234567-1234567",
-        "123-1234567-1234567",
-    ]
-    assert result["count"] == 6
+    assert result["order"] == ["123-1234567-1234567"] * 35
+    assert result["count"] == 35
 
     result = amazon_exception(mock_imap_amazon_exception, ["testemail@fakedomain.com"])
-    assert result["count"] == 7
+    assert result["count"] == 36
```

Comment:
```suggestion
    assert result["count"] == 8
```
This should be the amount of domains in the `AMAZON_DOMAINS` const plus the 1 extra added domain in the call.